test/parallel/test-util-inspect.js

```diff
@@ -2806,35 +2807,94 @@ assert.strictEqual(
 }
 
 {
+  const originalCWD = process.cwd();
+
+  process.cwd = () => (process.platform === 'win32' ?
+    'C:\\workspace\\node-test-binary-windows%20js-suites-%percent-encoded\\node' :
+    '/home/user%20directory/repository%encoded/node');
+
   // Use a fake stack to verify the expected colored outcome.
   const stack = [
     'TypedError: Wonderful message!',
-    '    at A.<anonymous> (/test/node_modules/foo/node_modules/bar/baz.js:2:7)',
+    '    at A.<anonymous> (/t/node_modules/foo/node_modules/bar/baz.js:2:7)',
```

Comment:
Any reason for this change? Not that it's a bad change, I'm just curious why it's necessary

Reply:
I honestly do not remember. I believe I wanted to reduce the output line length when rewriting the tests originally and decided to take a different approach later on. I'll change it back.